gratipay/renderers/jinja2_html.py

```diff
@@ -0,0 +1,25 @@
+import aspen_jinja2_renderer as base
+
+from markupsafe import escape as htmlescape
+
+
+class HTMLRenderer(base.Renderer):
+    def render_content(self, context):
+        # Extend to inject HTML-escaping i18n functions.
+        _, ngettext = context['_'], context['ngettext']
+        context['_'] = lambda *a, **kw: _(*a, **dict(kw, escape=htmlescape))
+        context['ngettext'] = lambda *a, **kw: ngettext(*a, **dict(kw, escape=htmlescape))
+        return base.Renderer.render_content(self, context)
```

Comment:
I don't see how we could upstream that in the future. OTOH inserting an `escape` function in the context could be upstreamed to `aspen-jinja2`, which is what I had in mind originally.